### DISCUSSION TITLE
PrivilegeCreateDatabase: permission deny to <new create user>

### DISCUSSION THREAD

**Discussion by CyanMystery** [milvus-io/milvus#33423] [Q&A and General discussion]
conn = connections.connect(host=MILVUS_HOST, port=19530, user='milvus3', password='123456')
 database = db.create_database("book")

raise error:
 <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.PERMISSION_DENIED
	details = "PrivilegeCreateDatabase: permission deny to milvus3"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"PrivilegeCreateDatabase: permission deny to milvus3

**Comment by yhmo** [comment]
Because you didn't grant "CreateDatabase" privilege to the role of "milvus3".
https://milvus.io/docs/users_and_roles.md
https://milvus.io/docs/rbac.md#4-Grant-a-privilege-to-a-role
